Tests/test_GraphicsChromosome.py

```diff
@@ -246,10 +246,9 @@ def test_widget(self):
         properties = new_stdout.getvalue()
         sys.stdout = save_stdout
 
-        self.assertTrue(
-            expected_string in properties,
-            "Unexpected results from dumpProperties: \n %s" % properties,
-        )
```

Comment:
Was there a merge conflict? That looks to have lost the black formatting...

Reply:
Yes, a merge conflict, fixed and re applied black; I'll try to fix the merge conflicts for this issue before the commit freeze for the new release; we are in lock down I can only do a tiny bit at a time.